### ISSUE TITLE
More than one field in tuple variants cause the macro to panic

### ISSUE THREAD

**Issue by AlexStrNik** [kdy1/is-macro#3]
Minimal reproducible example:
```rust
use is_macro::Is;

#[derive(Debug, Is)]
enum Foo {
    A(u8),
    B(u8, u8)
}

fn main() {
    println!("Hello, world!");
}
```

```bash
error[E0023]: this pattern has 1 field, but the corresponding tuple variant has 2 fields
 --> src\main.rs:3:17
  |
3 | #[derive(Debug, Is)]
  |                 ^^ expected 2 fields, found 1
...
6 |     B(u8, u8),
  |     --------- tuple variant defined here

error: aborting due to previous error
```

**Comment by snejugal** [comment]
Also errors if a tuple variant has zero fields:

```rust
use is_macro::Is;

#[derive(Debug, Is)]
enum Foo {
    A(),
}
```

```
$ cargo test --test tuples
   Compiling is-macro v0.1.0 (/home/snejugal/projects/is-macro)
error[E0023]: this pattern has 1 field, but the corresponding tuple variant has 0 fields
 --> tests/tuples.rs:3:17
  |
3 | #[derive(Debug, Is)]
  |                 ^^ expected 0 fields, found 1
4 | enum Foo {
5 |     A(),
  |     --- tuple variant defined here

error: aborting due to previous error
```

**Comment by kdy1** [comment]
@AlexStrNik @SnejUgal I fixed it and published `is-macro v0.1.3`
